lib/code_style_validation/plugin.rb

```diff
@@ -38,19 +44,31 @@ def check(config = {})
         raise 'Unknown SCM Provider'
       end
 
-      changes = get_changes(diff, ignore_file_patterns)
-      message = resolve_changes(changes)
+      changes = get_changes(diff, file_extensions, ignore_file_patterns)
+      offending_files, patches = resolve_changes(changes)
+
+      message = ''
+      unless offending_files.empty?
+        message = 'Code style violations detected in the following files:' + "\n"  
```

Comment:
Ooops... Sorry. I haven't yet reconfigured my Neovim properly after reinstalling the OS.